mirroroperator/registrymirror.py

```diff
@@ -450,7 +477,14 @@ def generate_stateful_set(self):
                                             "memory": "500Mi"}
                                 ),
                                 volume_mounts=volumes_to_mount,
-                            )
+                            ),
+                            client.V1Container(
+                                name="generate-ca-certs",
+                                image="octoblu/alpine-ca-certificates",
```

Comment:
I'm a little cautious of using this random image just to get the public trusted certs, as is @mikebryant 

maybe https://hub.docker.com/r/cloudbees/docker-certificates ?

Reply:
Yes, that is a good idea. I will use tag 1.2